classes/MyAdmin.php

```diff
@@ -14,6 +14,23 @@ class MyAdmin extends MyCommon
     /** @var MyTableAdmin */
     protected $TableAdmin;
                    
+    /** @var string to customize bootstrap.css path  */
+    protected $cssBootstrap = 'styles/bootstrap.css';
+    
+    /** @var string to customize font-awesome.css path */
+    protected $cssAwesome = 'styles/font-awesome.css';
+
+    /** @var array stylesheets added before end of head container */
+    protected $stylesheet = [];
```

Comment:
poznámka ke konzistenci:
* singular vs. plural. "stylesheet" vs "javascripts"
* z hlediska camelCase vs PascalCase vs všechna-malá - je to správně?